client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetails.tsx

```diff
@@ -134,13 +158,59 @@ const ClinicalDetails: React.FC = (): JSX.Element => {
                         כתובת לאשפוז ביתי:
                     </b>
                 </Typography>
+                <Autocomplete
```

Comment:
Why don't you use the common Autocomplete components that we have?

Reply:
לא התאים למה שעשיתי